### ISSUE TITLE
Ambiguous warning on console about unambiguous value matching

### ISSUE THREAD

**Issue by bryce-carson** [ashokkrish/spatialEpisim#28]
> Warning: Some values were not matched unambiguously:

The warning is clear in that some values should be listed after the warning message, but none are listed; this is ambiguous, which is ironic because the message includes the word "unambiguously".

**Comment by bryce-carson** [comment]
While refactoring cropBaseRasterHaxby.R this evening, I made a mistake when I called countrycode. This produced the warning this issue was opened for. The tongue-in-cheek reference to the ambiguity of this warning message must, therefore, have been due to a missing or NULL value rather than an incorrect one here, like `DRC`.

The issue is probably caused by a reactivity issue, where `countrycode` is called with an input value which is NULL at the time of call. This also makes sense for _when the warning is printed_, which is almost immediately after the application starts.

```R
countrycode("DRC", "iso3c", "country.name.en")
```

> [1] NA
> Warning message:
> Some values were not matched unambiguously: DRC
